galgebra/mv.py

```diff
@@ -79,10 +79,9 @@ def setup(ga):
         Mv.fmt = 1
 
         basis = [Mv(x, ga=ga) for x in ga.basis]
-        I = Mv(ga.iobj, ga=ga)  # default pseudoscalar
         x = Mv('XxXx', 'vector', ga=ga)  # testing vectors
         # return default basis vectors and grad vector if coords defined
-        return I, basis, x
+        return ga.mv_I, basis, x
```

Comment:
Any justification for this change? Change from local to indirect, contructing to reusing?

Reply:
Let's have that discussion in #79